Modules/binascii.c

```diff
@@ -334,14 +334,15 @@ binascii_a2b_uu_impl(PyObject *module, Py_buffer *data)
 binascii.b2a_uu
 
     data: Py_buffer
-    /
```

Comment:
Was this intended? If so, it needs documenting and testing, but it would be good to avoid if possible. Can’t you put a slash and asterisk after each other to have positional-only and then keyword-only parameters?